lte/gateway/python/setup.py

```diff
@@ -137,9 +137,9 @@
             # Keep grpcio and grpcio-tools on same version for now
```

Comment:
Is this comment outdated? To me it looks like grpcio-tools>=1.46.3 is used here and grpcio>=1.47.0 in orc8r/setup.py

Reply:
Was there a reason to use different versions? I assume we know that it works due to the tests but I would still prefer to keep them the same.

Reply:
I made the versions consistent